test/WebJobs.Script.Tests/NodeEndToEndTests.cs

```diff
@@ -712,6 +712,28 @@ public void ExcludedFunction_NotAddedToHost()
             await t;
         }
 
+        [Fact]
+        public async Task PromiseResolve()
+        {
+            JObject input = new JObject
+            {
+                { "scenario", "promiseResolve" }
+            };
+
+            Task t = Fixture.Host.CallAsync("Scenarios",
+                new Dictionary<string, object>()
+                {
+                    { "input", input.ToString() }
+                });
+
+            await TestHelpers.Await(() =>
```

Comment:
yep, modeling off of the nextTick test which is identical.  I'll make changes to both